osi_vehicle.proto

```diff
@@ -0,0 +1,692 @@
+            // This is the speed the function targets.
+            // E.g.: At the point of activation, the actual speed could be 80 km/h, 
+            // but the function tries to accelerate to 130 km/h.
+            //
+            // Unit: [km/h]
+            //
+            optional double targeted_speed = 2;
+
+            // The timegap describes the minimumdistance to the next vehicle in front.
+            //
+            // Unit: [s]
+            //
+            optional double timegap = 3;
+        }
+
+        //
+        // \brief Interface to describe the communication of an automated-driving function.
+        //
+        // Intentions, requests or wishes from the function to change the vehicle-state.
+        //
+        message Intentions
+        {
+            // All information about the trajectory the vehicle should follow.
+            //
+            // \note See osi_common_extension.
+            //
+            optional Trajectory trajectory = 1;
+
+            // Angle, angle-speed and torque.
+            //
+            // \note See osi_common_extension.
+            //
+            optional SteeringWheel steering_wheel = 2;
+
+            // Factor to scale the steeringtorque of the function output.
+            //
+            // \note 0 = no force of the function, 0.5 = half the force, 1 = 100% torque.
+            //
+            // Range: [0, 1] 
+            //
+            optional double steering_override_factor = 3;
+
+            // Acceleration-, brakepedal and clutch.
+            //
+            // \note See osi_common_extension.
+            //
+            optional Pedalry pedalry = 4;
+
+            // Position of the handbrake.
+            //
+            // \note A value of 0.0 means fully released and 1.0 means fully pressed.
+            //
+            // Range: [0, 1] 
+            //
+            optional double handbrake_position = 5;
+
+            // The light state of the vehicle.
+            //
+            optional MovingObject.VehicleClassification.LightState light_state = 6;
+
+            // Request that the driver has to take over.
+            //
+            // \note 0 = off; 1 = on.
+            //
+            optional bool driver_take_over_request = 7;
+
+            // Request to an ADAS-Function for a lane change.
+            //
+            optional LaneChangeRequest lane_change_request = 8;
+
+            // Request to an ADAS-Function for a lane change.
+            //
+            enum LaneChangeRequest
+            {
+                // Stay on the actual lane.
+                //
+                LANE_CHANGE_REQUEST_EGO_LANE = 0;
+
+                // Change to the left.
+                //
+                LANE_CHANGE_REQUEST_LC_LEFT = 1;
+
+                // Change to the right.
+                //
+                LANE_CHANGE_REQUEST_LC_RIGHT = 2;
+            }
+        }
+    }
+}
+
+
+//
+// \brief A description for the steering wheel.
+//
+message SteeringWheel
```

Comment:
This definition already exists in osi_ common, right?

Reply:
Really? I couldn't find it: https://github.com/OpenSimulationInterface/open-simulation-interface/blob/master/osi_common.proto
In the PR #309 I had it in the osi_common. , but I changed that.

Reply:
Now I saw that you are doint the PR on master and not on the other branch... You are right the master does not contain the definition!